scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -1027,31 +1007,37 @@ public void taskRestarted(final InternalJob job, final InternalTask task, final
             public Void executeWork(Session session) {
                 long jobId = jobId(job);
 
-                String jobUpdate = "update JobData set status = :status, "
-                        + "numberOfPendingTasks = :numberOfPendingTasks, "
-                        + "numberOfRunningTasks = :numberOfRunningTasks where id = :jobId";
+                String jobUpdate = "update JobData set status = :status, " +
+                    "numberOfPendingTasks = :numberOfPendingTasks, " +
+                    "numberOfRunningTasks = :numberOfRunningTasks, " +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "numberOfRunningTasks = :numberOfRunningTasks, " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)